.gitignore

```diff
@@ -102,3 +102,26 @@ dist
 
 # TernJS port file
 .tern-port
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
The `client/node_modules` folder is not ignored

Reply:
It is, because i also ignore `node_modules/`in the gitignore. I will delete the ´/node_modules`.